QEfficient/cloud/infer.py

```diff
@@ -60,7 +62,8 @@ def main(
         :local_model_dir (str): Path to custom model weights and config files. ``Defaults to None.``
         :cache_dir (str): Cache dir where downloaded HuggingFace files are stored. ``Defaults to None.``
         :hf_token (str): HuggingFace login token to access private repos. ``Defaults to None.``
-
+        :enable_qnn (bool): Enables QNN Compilation. ``Defaults to False.``
+        :qnn_config (str): QNN Config parameters file (if provided).
```

Comment:
Is this path of the QNN config parameters file, if yes please modify it like-
Path of QNN Config parameters file. ``Defaults is None.``